apps/api/src/main.ts

```diff
@@ -47,7 +47,7 @@ class CustomLogger implements LoggerService {
 async function bootstrap() {
   Sentry.init({
     dsn: process.env.SENTRY_DSN,
-    environment: process.env.NODE_ENV,
+    environment: process.env.NODE_ENV || 'development',
```

Comment:
In situations where the `NODE_ENV` is `e2e` or `test`, we would like to disable sentry. Hardcoding `development` in here would make that difficult.

Reply:
By default, the mode is served as `development`

Reply:
oh k.

Reply:
i would see how to disable sentry

Reply:
> In situations where the `NODE_ENV` is `e2e` or `test`, we would like to disable sentry. Hardcoding `development` in here would make that difficult.

done, with the latest commit